### PR TITLE
doc: add changelog for octopus release

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by theanalyst** [comment]
jenkins render doc

**Comment by theanalyst** [comment]
jenkins render docs

File: doc/releases/octopus.rst [hunk 1]
```diff
@@ -588,6 +588,3779 @@ Upgrade compatibility notes
...
+* cephadm: fix failure when getting keyring for deploying daemons (`pr#33679 <https://github.com/ceph/ceph/pull/33679>`_, Kiefer Chang)
+* cephadm: fix help message for bootstrap --mgr-id (`pr#32640 <https://github.com/ceph/ceph/pull/32640>`_, Sage Weil)
+* cephadm: fix inspect-image (`pr#33109 <https://github.com/ceph/ceph/pull/33109>`_, Sage Weil)
+* cephadm: fix logging defaults (`pr#32641 <https://github.com/ceph/ceph/pull/32641>`_, Sage Weil)
+* cephadm: fix name argument parsing during image check for non-ceph components (`pr#33114 <https://github.com/ceph/ceph/pull/33114>`_, Daniel-Pivonka)
+* cephadm: Fix Py3 ConfigParser deprecation warnings (`pr#32218 <https://github.com/ceph/ceph/pull/32218>`_, Michael Fritch)
+* cephadm: fix tox DeprecationWarning (`pr#32753 <https://github.com/ceph/ceph/pull/32753>`_, Michael Fritch)
+* cephadm: fix v1/v2 ip/addrv handling; explicitly check bind to ip:port (`pr#32392 <https://github.com/ceph/ceph/pull/32392>`_, Sage Weil)
+* cephadm: fix `alertmanager not implemented yet` (`pr#33694 <https://github.com/ceph/ceph/pull/33694>`_, Patrick Seid
```
Comment (smithfarm) [inline doc/releases/octopus.rst]: Golub (missing the closing "b")

Comment (smithfarm) [inline doc/releases/octopus.rst]: Golub (appears to be a global issue)

Comment (smithfarm) [inline doc/releases/octopus.rst]: lower-case Keithley

Comment (smithfarm) [inline doc/releases/octopus.rst]: real name? (nicks are supposed to not be allowed)

File: doc/releases/octopus.rst [hunk 2]
```diff
@@ -588,6 +588,3779 @@ Upgrade compatibility notes
...
+* build/ops,core: ceph-crash: use client.crash[.host] to post, and provsion keys via mgr/ssh + ceph-daemon (`pr#30734 <https://github.com/ceph/ceph/pull/30734>`_, Sage Weil)
+* build/ops,core: debian: fix ceph-mgr-modules-core files (`pr#33468 <https://github.com/ceph/ceph/pull/33468>`_, Sage Weil)
+* build/ops,core: os/bluestore: fix pmem osd build problem (`pr#28761 <https://github.com/ceph/ceph/pull/28761>`_, Peterson, Scott, Li, Xiaoyan)
+* build/ops,core: qa: stop testing on 16.04 xenial (`pr#28943 <https://github.com/ceph/ceph/pull/28943>`_, Sage Weil)
+* build/ops,mgr: mgr/diskprediction_local: Replaced old models and updated predictor (`pr#29437 <https://github.com/ceph/ceph/pull/29437>`_, Karanraj Chauhan)
+* build/ops,mgr: systemd: ceph-mgr: set MemoryDenyWriteExecute to false (`issue#39628 <http://tracker.ceph.com/issues/39628>`_, `pr#28023 <https://github.com/ceph/ceph/pull/28023>`_, Ricardo Dias)
+* build/ops,pybind: cmake, pybind: fix build on armhf (`pr#28843 <https://github.com/ceph/ceph/pull/28843>`_, Kefu Chai)
+* build/ops,rbd: rpm,deb: fix python dateutil module dependency (`pr#33624 <https://github.com/ceph/ceph/pull/33624>`_, Mykola Golu)
```
[the remark this answers is not in the document]
Comment (theanalyst) [reply doc/releases/octopus.rst]: interesting, thanks will get these

File: doc/releases/octopus.rst (hunk 1 above)
Comment (theanalyst) [reply doc/releases/octopus.rst]: isnt it the real name here?

**Comment by theanalyst** [comment]
sorry chose a wrong branch-name, recreated the pr https://github.com/ceph/ceph/pull/34130

**Pull request closed without merging**